lib/pbench/test/unit/server/test_server_audit.py

```diff
@@ -0,0 +1,121 @@
+        assert audits[0]["name"] == audits[1]["name"] == "first"
+        assert audits[0]["user_id"] == audits[1]["user_id"] == "1"
+        assert audits[0]["user_name"] == audits[1]["user_name"] == "test"
+        assert audits[2]["status"] == "BEGIN"
+        assert audits[3]["status"] == "FAILURE"
+        assert audits[2]["operation"] == audits[3]["operation"] == "UPDATE"
+        assert audits[2]["name"] == audits[3]["name"] == "second"
+        assert audits[2]["user_id"] == audits[3]["user_id"] is None
+        assert audits[2]["user_name"] == audits[3]["user_name"] == "fake"
+
+    def test_get_name(self, query_get, make_audits):
+        """Get all audit records matching a specific operation name"""
+        response = query_get(params={"name": "first"}, expected_status=HTTPStatus.OK)
+        audits = response.json
+        assert len(audits) == 2
+        assert audits[0]["status"] == "BEGIN"
+        assert audits[1]["status"] == "SUCCESS"
+        assert audits[0]["operation"] == audits[1]["operation"] == "CREATE"
+        assert audits[0]["name"] == audits[1]["name"] == "first"
+        assert audits[0]["user_id"] == audits[1]["user_id"] == "1"
+        assert audits[0]["user_name"] == audits[1]["user_name"] == "test"
+
+    def test_get_operation(self, query_get, make_audits):
+        """Get all audit records matching a specific operation"""
+        response = query_get(
+            params={"operation": "CREATE"}, expected_status=HTTPStatus.OK
+        )
+        audits = response.json
+        assert len(audits) == 2
+        assert audits[0]["status"] == "BEGIN"
+        assert audits[1]["status"] == "SUCCESS"
+        assert audits[0]["operation"] == audits[1]["operation"] == "CREATE"
+        assert audits[0]["name"] == audits[1]["name"] == "first"
+        assert audits[0]["user_id"] == audits[1]["user_id"] == "1"
+        assert audits[0]["user_name"] == audits[1]["user_name"] == "test"
+
+    def test_get_status_begin(self, query_get, make_audits):
+        """Get all audit records matching a specific status"""
+        response = query_get(params={"status": "BEGIN"}, expected_status=HTTPStatus.OK)
+        audits = response.json
+        assert len(audits) == 2
+        assert audits[0]["status"] == audits[1]["status"] == "BEGIN"
+        assert audits[0]["operation"] == "CREATE"
+        assert audits[1]["operation"] == "UPDATE"
+        assert audits[0]["name"] == "first"
+        assert audits[1]["name"] == "second"
+
+    def test_get_status_failure(self, query_get, make_audits):
+        """Get all audit records showing a failure"""
+        response = query_get(
+            params={"status": "FAILURE"}, expected_status=HTTPStatus.OK
+        )
+        audits = response.json
+        assert len(audits) == 1
+        assert audits[0]["status"] == "FAILURE"
+        assert audits[0]["operation"] == "UPDATE"
+        assert audits[0]["name"] == "second"
```

Comment:
This test seems redundant with the previous one (i.e., they both query on `"status"`).

On the other hand, shouldn't you have a test or three ("before", "between", and "after") based on timestamp?